User Interface/RPG Interface/source/State_Title.hx

```diff
@@ -66,6 +66,15 @@ class State_Title extends FlxUIState
 		}		
 	}	
 	
+	public override function update(elapsed:Float):Void
```

Comment:
Is this just debug code, or meant to stay?

You can just type `vc` in the console btw.